pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "d-fast_morphological_impact"
+version = "2.0.8"
+description = "A tool to perform a morphological impact analysis based on a number of D-Flow FM simulations."
+authors = ["Stichting Deltares <delft3d.support@deltares.nl>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
```

Comment:
Is het een idee om dit te sorteren en gelijk te trekken tussen BE/MI, zodat de verschillende packages onderop staan.
Of om dit in tweeën te delen met een specifieke [tool.poetry.dependencies] voor BE en MI.
E.g. Generic D-Fast & Specific BE/MI.

Reply:
Het lijkt me in ieder geval een goed idee om te weten wat de resterende verschillen zijn in requirements tussen MI en BE; sorteren van de requirements helpt daarbij. MI maakte in eerste instantie geen figuren, maar ondertussen wel. BE doet waarschijnlijk nog iets meer met geospatial analyse ...